modelscope/cli/download.py

```diff
@@ -24,11 +26,17 @@ def define_args(parsers: ArgumentParser):
         """ define args for download command.
         """
         parser: ArgumentParser = parsers.add_parser(DownloadCMD.name)
-        parser.add_argument(
+        group = parser.add_mutually_exclusive_group(required=True)
+        group.add_argument(
             '--model',
             type=str,
-            required=True,
-            help='The model id to be downloaded.')
+            help='The model id to be downloaded, model or dataset must provide.'
+        )
+        group.add_argument(
+            '--dataset',
+            type=str,
+            help=
+            'The dataset id to be downloaded, model or dataset must provide.')
```

Comment:
The id of the dataset to be downloaded. For download, the id of either a model or dataset must be provided.

Reply:
updated